cmd/openshift-install/create.go

```diff
@@ -249,7 +249,8 @@ func logComplete(directory string) error {
 		return err
 	}
 	kubeconfig := filepath.Join(absDir, "auth", "kubeconfig")
-	logrus.Infof("Install complete! Run 'export KUBECONFIG=%s' to manage your cluster.", kubeconfig)
-	logrus.Info("After exporting your kubeconfig, run 'oc -h' for a list of OpenShift client commands.")
+	// TODO: Direct users to web-console
+	// TODO: Get kubeadmin password, log here
```

Comment:
we already give the user admin kubeconfig. I rather tell people how to get/create the password.

Reply:
> I rather tell people how to get/create the password.

How about we point folks at their admin kubeconfig (like we do now) and then drop a link to a "common day-2 operations" landing page?  Where would something like that live?  Master branch of https://github.com/openshift/openshift-docs ?  Some sort of pre-release branch?  Some other repo?

Reply:
without this:  openshift/origin#21580  in order to have a user that can login to the console/prometheus/etc, you'd have to set up an identity provider.  This kubeadmin user will be a magic user, expected to be removed upon setting up your idp.

Reply:
> We need to be able to do oauth flows for the console, etc. and that can't be done with just a kubeconfig.

But this PR is just pushing in a new secret, right?  That's not installer-specific.  At any time after the control plane comes up, you can use the admin kubeconfig and `oc` to push in the `kubeadmin-password` secret.  You could wrap it up in an `oc` subcommand if constructing the password, hashing it, and putting it inside a secret were a concern.  A benefit of that approach is that it would be opt-in, folks who are going to start setting up a fully-fledged identity provider immediately after the cluster comes up could skip it and have nothing around to clean up.